recipes/boost/all/conanfile.py

```diff
@@ -53,6 +53,7 @@ class BoostConan(ConanFile):
         "lzma": [True, False],
         "zstd": [True, False],
         "segmented_stacks": [True, False],
+        "debug_level": [i for i in range(1, 14)],
```

Comment:
@danimtb not sure, but CCI doesn't lile the syntax:
```
"debug_level": range(1, 14),
```
it throws:
```
ERROR: Object of type range is not JSON serializable
```
maybe there is a better/more elegant way to initialize?
also, maybe it's CCI or conan bug that it doesn't accept the range? (locally it works)

Reply:
opened https://github.com/conan-io/conan-center-index/issues/954